modules/ROOT/pages/distributed-transactions-cxx-release-notes.adoc

```diff
@@ -45,16 +45,25 @@ $ sudo dpkg -i couchbase-transactions*.deb
 
 // 1.0.0 release notes here:
 [#latest-release]
+== Version 1.0.0 (19 April 2021)
+
+* Supports couchbase server 6.6.1 and later.
+* Transactions for KV operations only (no query transactions yet)
+* Synchronous kv transaction operations (no async transaction operations yet)
+* Minimal c++ client included.
+
+=== Known issues
+
 == Version 1.0.0 Beta 1 (13 October 2020)
 
 First beta release of 1.0.0
 
 
 === Known Issues
 
-* Owing to https://issues.couchbase.com/browse/MB-41944[MB-41944], this release _should not be used together with a deployment using Sync Gateway_.  
-Owing to this issue there is also a known theoretical data loss problem.  
-This release (as with all beta releases) should not be used in production.  
+* Owing to https://issues.couchbase.com/browse/MB-41944[MB-41944], this release _should not be used together with a deployment using Sync Gateway_.
+Owing to this issue there is also a known theoretical data loss problem.
+This release (as with all beta releases) should not be used in production.
```

Comment:
Need to remove this line^

Reply:
That's in the known issues section for the beta release, not the 1.0.0 known issues.  I think it is ok.

Reply:
But I'm happy to remove it

Reply:
Ohhh, sorry about that.  I see now.